docs/Getting Started.md

```diff
@@ -0,0 +1,92 @@
+# Getting Started
+Welcome to Inductiva, a Python API package that simplifies running large-scale 
+physical system simulations in the cloud. Just few lines of code unleash its potential, freeing 
+you from local hardware limits and the technical hassles of scaling simulations. 
+
+### What We'll Cover:
+
+* [Inductiva API Installation]()
+* [What to read next]()
+* [Troubleshooting]()
+* [Frequently Asked Questions (FAQs)]()
+* [Useful Terminology]()
+
+## Inductiva API Installation
+
+To install the latest Inductiva package release on PyPI with pip, just open your 
+terminal and run:
+
+```
+pip install --upgrade inductiva
+```
+---
+
+> **Tip:** To ensure smooth installation and operation of our API, we recommend 
+keeping your Python package manager, Pip, up-to-date. Visit [Pip's official installation guide](https://pip.pypa.io/en/stable/installation/) for detailed instructions on updating Pip.
+
+
+Encountering issues? Don’t worry. Head over to [Troubleshooting]() to work it out.
+
+Otherwise, you're all set! You can now decide [what to read next]().
+
+## What to read next
+
+You can start experimenting with Inductiva by choosing from a variety of ready-to-use, 
+[open-source simulators]() that are immediately available for use on our default 
+machines.
+
+You can set up custom, dedicated machine groups for more complex or [larger-scale simulation]() 
```

Comment:
So, this is about "Choosing your Hardware" or maybe about "Scaling your Simulation"

The title here should be an action point for the user.